### DISCUSSION TITLE
Can you integrate this with contacts app

### DISCUSSION THREAD

**Discussion by Waqaralamlive** [saber-notes/saber#60] [Ideas]
Assalamualaikum brother

can you integrate saber notes with the contacts app where i can save hand writing notes for each contacts ?

and one more thing adding image function is not working in your app

thanks in advance

**Comment by adil192** [comment]
Honestly I can't see that being a commonly used mechanism, and I'd like to keep the amount of data the app accesses to a minimum.
And yes that's not been implemented yet. You can see the roadmap [here](https://github.com/adil192/saber/discussions/1)

**Reply by Waqaralamlive** [reply]
But i need that kind of contact application 
If you want i can pay you

**Reply by adil192** [reply]
I'm sorry but it seems you might be looking for a different app to Saber. This is primarily intended to replace physical notebooks.
